src/utilities/strategy-helpers/smi-canary-deployment-helper.ts

```diff
@@ -90,7 +92,7 @@ function createCanaryService(kubectl: Kubectl, filePaths: string[]) {
                     newObjectsList.push(newStableServiceObject);
 
                     core.debug('Creating the traffic object for service: ' + name);
-                    const trafficObject = createTrafficSplitManifestFile(name, 0, 0, 1000);
+                    const trafficObject = createTrafficSplitManifestFile(kubectl, name, 0, 0, 1000);

```

Comment:
can we keep the constants in CAPs.

Reply:
This is just a variable name. I think we can keep it like this, just like other consts.